examples/bitmap_operation/README.md

```diff
@@ -1,15 +1,26 @@
-# How to compile    
```

Comment:
We have to have 4 whitespaces at the end of needed lines in **md** file, otherwise file view is corrupted